lib/vspc/handlers.go

```diff
@@ -0,0 +1,204 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vspc
+
+import (
+	"bytes"
+	"crypto/rand"
+	"fmt"
+	"io"
+	"net"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+
+	"github.com/vmware/vic/pkg/telnet"
+	"github.com/vmware/vic/pkg/trace"
+)
+
+// handler is the handler struct for the vspc
+type handler struct {
+	vspc *Vspc
```

Comment:
[cosmetic] you could embed *Vspc directly which simplifies h.vspc.XYZ type calls